server/base/src/main/java/org/apache/accumulo/server/master/LiveTServerSet.java

```diff
@@ -120,11 +120,17 @@ public TabletServerStatus getTableMap(boolean usePooledConnection) throws TExcep
       if (usePooledConnection == true)
         throw new UnsupportedOperationException();
 
+      long start = System.currentTimeMillis();
```

Comment:
I've started preferring to use nanoTime() for small, single-threaded timings. I know that's the sort of thing that it's meant to do well (local timings), but I'm not sure if there's a downside to it. Just thought I'd mention it. Like you say, at the end of the day, it probably won't matter.

I think @keith-turner had originally turned me onto it.